<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationsForm</class>
 <widget class="QWidget" name="OperationsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="4">
    <widget class="QCheckBox" name="applyToName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>name</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="applyLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Apply to:</string>
     </property>
     <property name="indent">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="addOperation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Add Operation</string>
      </property>
      <property name="icon">
       <iconset theme="list-add">
        <normaloff/>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Insert</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="operationsTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="moveOpDown">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="go-down">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="moveOpUp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="go-up">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="5">
    <widget class="QCheckBox" name="applyToExtension">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>extension</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QComboBox" name="deleteOperation">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Delete</string>
      </property>
      <property name="icon">
       <iconset theme="edit-delete">
        <normaloff/>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Delete All</string>
      </property>
      <property name="icon">
       <iconset theme="edit-clear">
        <normaloff/>
       </iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1" colspan="7">
    <widget class="QStackedWidget" name="operationsView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
